concent_api/core/validation.py

```diff
@@ -293,6 +293,21 @@ def validate_positive_integer_value(value):
         )
 
 
+def validate_scene_file(scene_file):
+    if not scene_file.startswith(('\\', '/')):
```

Comment:
I would not assume this...
@cameel ?

Reply:
I do not know if this is a valid assumption or not. We would have to find this in Golem's source to be sure. Maybe ask on Rocket?